dev-docs/modules/iabCatalogTranslation.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page_v2
+page_type: module
+title: Module - IAB Category Translation
+description: Converts between ad agency brand categories and IAB brand categories.
+module_code : CategoryTranslation
+display_name : CategoryTranslation
+enable_download : true
+sidebarType : 1
+---
+
+# IAB Category Translation
+
+{:.no_toc}
+
+This module supports the mapping between ad agency brand categories and IAB subcategories and conversion of IAB subcategories to Freewheel industry group identifiers. 
+
+Each bid request must return one [IAB subcategory](https://support.aerserv.com/hc/en-us/articles/207148516-List-of-IAB-Categories).
+
+The module provides the following: 
+
+1. Provides an API that converts the demand providers own ad categories into IAB subcategories. If a demand provider elects to use Prebid API for this process, they must include a path to a local mapping file that converts the ad server categories to IAB subcatgories. 
```

Comment:
1 can be removed. This module only does point 2

Reply:
Revised doc.